index.html

```diff
@@ -36,5 +36,104 @@ <h1>I'm Norman<br> Glad to see you!</h1>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a class="project-btn" href="">See Project</a>
+            </div>
+        </div>
+        <div class="work-card">
+            <div class="work-image">
+                <img src="images/work2.png" alt="work 2">
+            </div>
+            <div class="work-details">
+                <h2>Multi-Post<br> Stories</h2>
+                <p class="work-subtext">CANOPY<span class="dot"></span><span class="description">Back End Dev</span><span class="dot"></span><span class="description">2015</span> </p>
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a class="project-btn" href="">See Project</a>
+            </div>
+        </div>        
+        <div class="work-card">
+            <div class="work-image">
+                <img src="images/work3.png" alt="work 3">
+            </div>
+            <div class="work-details">
+                <h2>Tonic</h2>
+                <p class="work-subtext">CANOPY<span class="dot"></span><span class="description">Back End Dev</span><span class="dot"></span><span class="description">2015</span> </p>
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a class="project-btn" href="">See Project</a>
+            </div>
+        </div>        
+        <div class="work-card">
+            <div class="work-image">
+                <img src="images/work4.png" alt="work 4">
+            </div>
+            <div class="work-details">
+                <h2>Multi-Post<br> Stories</h2>
+                <p class="work-subtext">CANOPY<span class="dot"></span><span class="description">Back End Dev</span><span class="dot"></span><span class="description">2015</span> </p>
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a class="project-btn" href="">See Project</a>
+            </div>
+        </div>        
+    </section>
+    <section class="contact-section">
+        <div class="primary-text">
+            <h1 class="primary-text-change">About<br>Myself</h1>

```

Comment:
- [OPTIONAL ] I suggest removing the `<h1>` tag from here too and using another header tag `hx` (eg `<h2>`,  `<h3>`, ...) instead. to Follow best practices for HTML. **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/). 👌